text/0000-drop-types-in-const.md

```diff
@@ -0,0 +1,68 @@
+- Feature Name: `drop_types_in_const`
+- Start Date: 2016-01-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Allow types with destructors to be used in `static` items and in `const` functions, as long as the destructor never needs to run in const context.
+
+# Motivation
+[motivation]: #motivation
+
+Most collection types do not allocate any memory when constructed empty. With the change to make leaking safe, the restriction on `static` items with destructors
+is no longer required to be a hard error.
+
+Allowing types with destructors to be directly used in `const` functions and stored in `static`s will remove the need to have
+runtime-initialisation for global variables.
+
+# Detailed design
```

Comment:
It may not have been clear enough here - This is just to use the current `const` structure (no life before main, no life after it) to do compile-time construction of objects that support it.